.changes/next-release/feature-8ae7facb-e1c5-4f25-9955-8c36b3cf59c4.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Added the ability to copy the URL to an S3 object"
```

Comment:
Should we standardize our descriptions? I can never remember if we're trying to make everything past or present tense

Reply:
Ya "Add" does sound better